crates/astria-sequencer/src/grpc/sequencer.rs

```diff
@@ -84,15 +84,16 @@ impl SequencerService for SequencerServer {
             ));
         }
 
-        let mut rollup_ids: Vec<RollupId> = vec![];
-        for id in request.rollup_ids {
-            let Ok(rollup_id) = RollupId::try_from_vec(id) else {
-                return Err(Status::invalid_argument(
-                    "invalid rollup ID; must be 32 bytes",
-                ));
-            };
-            rollup_ids.push(rollup_id);
-        }
+        let Ok(rollup_ids) = request
```

Comment:
note this is secondary, but updated to match the style of the rest of the code here closer.